src/Common/src/Interop/Windows/kernel32/Interop.GetFileAttributesEx.cs

```diff
@@ -68,10 +67,13 @@ internal unsafe struct WIN32_FIND_DATA
             internal uint nFileSizeLow;
             internal uint dwReserved0;
             internal uint dwReserved1;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 260)]
-            internal string cFileName;
-            [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 14)]
-            internal string cAlternateFileName;
+            private fixed char _cFileName[260];
+            private fixed char _cAlternateFileName[14];
+
+            internal Span<char> cFileName
+            {
+                get { fixed (char* c = _cFileName) return new Span<char>(c, 260); }
+            }
```

Comment:
Note that this `Span<char>`-based approach needs to be used with care.  With the span implementation in coreclr, this will be safe if this struct is on the heap (e.g. as a field of some class), and it'll be safe for the immediate caller that has this struct as a local to use this method, but if that caller returned this span, its caller would end up with a span pointing to reclaimed stack space that's no longer valid to be used.